spec/requests/api/v1/registrations/upload_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'rails_helper'
```

Comment:
I think this file supposed to be named `update_spec.rb`, because it is testing the `update` method in the registrations controller